### ISSUE TITLE
404 Not found

### ISSUE THREAD

**Issue by schniggie** [m1nl/pompa-docker#1]
Hi,

just deployed all containers and tried to access port 8082. I only get a "404 Not found". Tried it on the exposed docker network and also tried a SSH forward to -L8082:localhost:8082 on the docker host, itself. But that hasn't changed anything.

Attempt #1: http:<Public_Docker-Host_IP>:8082
```
app_1     | I, [2020-06-07T08:18:15.832601 #184]  INFO -- : [c9faaaa9-cbfd-419f-a8dd-4fc04ed27665] Started GET "/api/public/" for 109.192.199.208 at 2020-06-07 08:18:15 +0000
app_1     | I, [2020-06-07T08:18:15.833351 #184]  INFO -- : [c9faaaa9-cbfd-419f-a8dd-4fc04ed27665] Processing by PublicController#index as HTML
app_1     | I, [2020-06-07T08:18:15.833774 #184]  INFO -- : [c9faaaa9-cbfd-419f-a8dd-4fc04ed27665] Completed 404 Not Found in 0ms (Allocations: 107)
facade_1  | 109.192.199.208 - - [07/Jun/2020:08:18:15 +0000] "GET / HTTP/1.1" 404 819 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36" "-"
```

Attempt #2: http:localhost:8082 with SSH -L 8082:localhost:8082
```
app_1     | I, [2020-06-07T08:19:53.038591 #184]  INFO -- : [f06d59af-8c52-4562-9b31-1488e137f1b8] Started GET "/api/public/" for 172.21.0.5 at 2020-06-07 08:19:53 +0000
app_1     | I, [2020-06-07T08:19:53.039265 #184]  INFO -- : [f06d59af-8c52-4562-9b31-1488e137f1b8] Processing by PublicController#index as HTML
app_1     | I, [2020-06-07T08:19:53.039648 #184]  INFO -- : [f06d59af-8c52-4562-9b31-1488e137f1b8] Completed 404 Not Found in 0ms (Allocations: 104)
facade_1  | 172.21.0.1 - - [07/Jun/2020:08:19:53 +0000] "GET / HTTP/1.1" 404 819 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36" "-"
```

And btw. there is no /web folder according to the README, there should also be some configs.

Thanks for your help :)

**Comment by m1nl** [comment]
It seems that README got a bit outdated... sorry for that - fixed in 4e9b3493c82e8a0e9f14f8d1aaf1bce113d92c5c .

- web folder has been renamed to facade
- administrative interface is exposed on 8081 port

**Comment by m1nl** [comment]
Okay, I've corrected all references to 8081 / 8082 ports in 056272944a4f99adb8b9c07f6fb885aafd71a41c
